controlled-vocabulary/src/main/resources/META-INF/drlineconfiguration.json

```diff
@@ -1500,6 +1500,17 @@
 			}
 		]
 	},
+	{
+		"name": "AntiFam",
+		"displayName": "AntiFam",
+		"category": "FMD",
+		"uriLink": "https://www.ebi.ac.uk/interpro/entry/antifam/%id",
+		"attributes": [
```

Comment:
Dushi example on TRM-31255 using %s for id in uriLink and we are using %id. Is it okay?